app/controllers/reports/progress_controller.rb

```diff
@@ -0,0 +1,45 @@
+# This emulates what progress tab shows in the API, but does it within our dashboard.
+# The main purpose currently is to make it easier to develop on progress tab in dev.
+class Reports::ProgressController < AdminController
```

Comment:
This is mainly for me and @claudiovallejo to make progress tab work easier...though I imagine eventually it could be useful for admins.

For now I'll block it in prod via a feature flag or similiar.